src/components/shared/CallNodeContextMenu.js

```diff
@@ -671,16 +671,16 @@ function TransformMenuItem(props: {|
   +title: string,
 |}) {
   return (
-    <MenuItem onClick={props.onClick} data={{ type: props.transform }}>
-      <div className="callNodeContextMenuWithKey" title={props.title}>
-        <div className="callNodeContextMenuWithKeyText">
```

Comment:
I removed these 2 divs that were quite unnecessary and make things more difficult to style.